megatron/training/arguments.py

```diff
@@ -2706,6 +2706,8 @@ def _add_distributed_args(parser):
                        help='If set, enable full sharding in megatron-fsdp Hybrid Sharded Data Parallel (HSDP) mode.')
     group.add_argument('--num-distributed-optimizer-instances', type=int, default=1,
                        help='Number of Distributed Optimizer copies across Data Parallel domain.')
+    group.add_argument('--no-mfsdp-comm', action='store_true',
```

Comment:
Consider using argparse.BooleanOptionalAction so a use can explicitly opt in too